MediaManager/Platforms/Ios/Video/VideoView.cs

```diff
@@ -108,13 +108,5 @@ public VideoAspectMode VideoAspect
             get => PlayerViewController.ShowsPlaybackControls;
             set => PlayerViewController.ShowsPlaybackControls = value;
         }
-
-        protected override void Dispose(bool disposing)
```

Comment:
This should not be removed?

Reply:
It caused a crash after a few videos play in Xamarin.Forms. In native is not called at all.

Reply:
Ok, but then maybe we need to move this function to some different method.

Reply:
Do you have any idea? What was the purpose of that?

Reply:
I moved it to the `PlayerViewController`

Reply:
I moved it to `PlayerViewController`,